src/darwin/Framework/CHIPTests/MTROTAProviderTests.m

```diff
@@ -622,8 +643,8 @@ - (void)test004_DoBDXTransferDenyUpdateRequest
     while (fakeImage.length < rawImageSize) {
         [fakeImage appendData:rawImagePiece];
     }
-    NSString * rawImagePath = @"/tmp/test004-raw-image";
-    NSString * imagePath = @"/tmp/test004-image";
+    NSString * rawImagePath = @"/tmp/test004-raw-image-test";
+    NSString * imagePath = @"/tmp/test004-image-test";
```

Comment:
Why the extra "-test" suffixes here?

Reply:
not needed. since you renamed the other to test005

Reply:
done.